src/main/java/at/hannibal2/skyhanni/data/model/SkyblockStat.kt

```diff
@@ -10,6 +10,8 @@ enum class SkyblockStat(val icon: String) {
     DEFENSE("§a❈"),
     STRENGTH("§c❁"),
     INTELLIGENCE("§b✎"),
+
+    // TODO rename to CRITICAL
```

Comment:
i dont really see a reason why this is needed

Reply:
Yeah, the official stat names are "Crit Damage" and "Crit Chance".

Reply:
then definitely shouldnt be changed